docs/advanced-features/package-events.md

```diff
@@ -0,0 +1,105 @@
+# Package events
+
+Every time a package revision is created in one of the Quilt stack buckets, the
+Quilt stack emits event to the default
+[EventBridge](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html)
+bus. The event has the following structure:
```

Comment:
Each time a package is created or revised in a Quilt stack bucket,
the stack emits a `package-revision` event on the default
[EventBridge](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html) bus.
Events have the following structure: